### PR TITLE
Add Mirador Image Tools plugin

### LINKED ISSUE #1350: Add rotate plugin to Mirador viewer
Several users requested the ability to rotate images displayed in Mirador. This plugin exists but is not implemented in DLME.

### PR DESCRIPTION
**Pull request by marlo-longley**
Fixes #1350 

- Adds plugin to package.json
- Enables plugin in config
- Checks for plugin existence in spec

### CODE REVIEW

File: app/javascript/packs/mirador.js [hunk 1]
```diff
@@ -52,7 +54,15 @@ Mirador.viewer({
   workspaceControlPanel: {
     enabled: false,
   },
-  windows: [
-    {loadedManifest: manifestUrl}
-  ]
-})
+  windows: [{
+    imageToolsEnabled: true,
+    imageToolsOpen: true,
```
Comment (cbeer) [inline app/javascript/packs/mirador.js:59]: @ggeisler : should we default to showing the whole toolbar or just the toggle? I'm not sure what the intended use for these tools is..

Comment (marlo-longley) [reply app/javascript/packs/mirador.js:59]: Screen Shot 2022-02-25 at 12 00 31

Current view for reference with "open" as default

Comment (marlo-longley) [reply app/javascript/packs/mirador.js:59]: Vs. closed default
Screen Shot 2022-02-25 at 12 01 20
t

Comment (ggeisler) [reply app/javascript/packs/mirador.js:59]: I suggest closed by default. I think the goal for DLME is to enable users who might have a use case for one or more of the tools to have access to them, but there are likely far more DLME users who don't have image-tool-related use cases. So the more unobtrusive closed version seems appropriate as the default.

**Review by thatbudakguy — APPROVED** [review]

**Pull request merged**